berkeley-mobile/Utils/UIViewController+Extensions.swift

```diff
@@ -28,22 +28,34 @@ extension UIViewController {
     
     // Presents an alert with multiple options and completion handler
     public func presentAlertLinkUrl(title: String, message: String, options: String..., website_url: URL) {
-        let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        
+        var alertVC: AlertView!
+        
```

Comment:
I think this function is meant to present any number of options and it loops through all of them, adding them to the alert.
Because your AlertView doesn't support this (and we don't use that in the app), you should probably change this function to only take in title, message, and a single button text option.
Then, get rid of the loop and just present your AlertView with the given title, message, a cancel button, and the other button that opens the url.

Reply:
the function here is`presentAlertLinkUrl`